src/App.scss

```diff
@@ -0,0 +1,3 @@
+.Test {
```

Comment:
<!--
  0 failure: 
  1 warning:  Selector should b...
  
  
  DangerID: danger-id-Danger;
  File: src/App.scss;
  Line: 1;
-->

- :warning: Selector should be written in lowercase with hyphens (selector-class-pattern)